INSTALL.sh

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env bash
 
-# First install the configuration files:
+# Important: The script should check if DKMS is Present, because its important so no one would cry at issues
+CHECK=$(command -v dkms 2>/dev/null)
+if [ -n "$CHECK" ]
+  then 
+  echo "DKMS Present, Istalling Anbox Modules"
+  else
+  echo "DKMS Did not respond, it seems that DKMS is not Present or you are not running in sudo/root mode"
+  echo "To make sure, please install dkms and try running the script again with sudo"
```

Comment:
```suggestion
if ! [ -n "$CHECK" ]
  then
  >&2 echo "DKMS command not found, it seems that DKMS is not installed or you do not have permission to use it."
  >&2 echo "Please install the dkms package from your distribution vendor and re-run this tool as root."
```

I don't think we need to explicitly state that we found DKMS, don't mind the quick reword